lib/osf-components/addon/components/sign-up-form/template.hbs

```diff
@@ -52,33 +51,29 @@
             </div>
         </div>
     </div>
-    {{! template-lint-disable no-implicit-this }}
-    {{#validated-input/checkbox
+    <ValidatedInput::Checkbox
         data-test-sign-up-tos
-        model=this.userRegistration
-        valuePath='acceptedTermsOfService'
-        shouldShowMessages=this.didValidate
-        disabled=this.hasSubmitted
         local-class='tos-checkbox'
-    }}
+        disabled={{this.hasSubmitted}}
+        @model={{this.userRegistration}}
+        @valuePath='acceptedTermsOfService'
+        @shouldShowMessages={{this.didValidate}}
+    >
         <SignUpPolicy />
-    {{/validated-input/checkbox}}
-    {{! template-lint-enable no-implicit-this }}
+    </ValidatedInput::Checkbox>
     {{#if this.hasSubmitted}}
         <div class='osf-box-lt' local-class='help-block'>
             <p class='ext-success p-xs'>{{t 'osf-components.sign-up-form.registration_success' email=this.userRegistration.email1}}</p>
         </div>
     {{else}}
         <div class='col-md-12' local-class='RecaptchaParent'>
-            {{! template-lint-disable no-implicit-this }}
-            {{validated-input/recaptcha
+            <ValidatedInput::Recaptcha
                 data-test-recaptcha
-                model=this.userRegistration
-                valuePath='recaptchaResponse'
-                shouldShowMessages=this.didValidate
-                bindReset=(action (mut this.resetRecaptcha))
-            }}
-            {{! template-lint-enable no-implicit-this }}
+                @model={{this.userRegistration}}
+                @valuePath='recaptchaResponse'
```

Comment:
Can we always use `{{}}`? For example, can this be `{{'recaptchaResponse'}}`?

Reply:
`{{}}` are not needed here because `valuePath` is coerced to string. Basically, `{{}}` are needed for boolean and numbers so they don't get coerced to string unintentionally.

Reply:
I don't like how we have to think what kind of argument we are passing before choosing to use `{{}}` or not. If using `{{}}` on strings doesn't affect anything, might as well make it so that we always use `{{}}` regardless of argument types. We've seen enough confusion on when and when not to use `{{}}` already.

Reply:
This is beyond scope. Prefer we decide together as a team on this. Will make a note of this.